focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -68,34 +68,66 @@ DefaultDelegates.pointTemplates = {
 --]]
 function DefaultDelegates.getAfPoints(photo, metaData)
   local focusPoint = ExifUtils.findFirstMatchingValue(metaData, DefaultDelegates.metaKeyAfPointUsed)
+  local result = nil, nil
 
+  -- If the typical AF point data is not present, check for an absolute value
   if focusPoint == nil then
-    LrErrors.throwUserError("Unable to find any AF point info within the file.")
-    return nil, nil
-  end
-
-  if DefaultDelegates.focusPointsMap[focusPoint] == nil then
-    LrErrors.throwUserError("The AF-Point " .. focusPoint .. " could not be found within the file.")
-    return nil, nil
-  end
-
-  -- TODO: The addition of the dimension should be removed once all config files have been
-  -- updated to reflect the center of the focus points
-  local x = DefaultDelegates.focusPointsMap[focusPoint][1] + (.5 * DefaultDelegates.focusPointDimen[1])
-  local y = DefaultDelegates.focusPointsMap[focusPoint][2] + (.5 * DefaultDelegates.focusPointDimen[2])
-
-  local result = {
-    pointTemplates = DefaultDelegates.pointTemplates,
-    points = {
-      {
-        pointType = DefaultDelegates.POINTTYPE_AF_SELECTED_INFOCUS,
-        x = x,
-        y = y,
-        width = DefaultDelegates.focusPointDimen[1],
-        height = DefaultDelegates.focusPointDimen[2]
+    local afAreaXPosition = ExifUtils.findFirstMatchingValue(metaData, {"AF Area X Position"})
```

Comment:
This is nice work. Can you provider a sample image with the camera turned 90 CW and then 90 CCW? Often times camera will invert the coords and I want to see if we need to account for that.